src/jit/codegenarm64.cpp

```diff
@@ -3297,24 +3296,59 @@ void CodeGen::genCodeForInitBlkUnroll(GenTreeBlk* initBlkNode)
         initVal = initVal->gtGetOp1();
     }
 
-    assert(!dstAddr->isContained());
-    assert(!initVal->isContained());
+    assert(dstAddr->isUsedFromReg());
+    assert(initVal->isUsedFromReg() || (initVal->IsIntegralConst(0) && ((size & 0xf) == 0)));
```

Comment:
I'm not sure why this assert has a (size & 0xf) component.
You could try to use the ZR (fixed zero) register in the code below (since that is the common case.  But I don't see that in the code below.